src/views/index.njk

```diff
@@ -1,21 +1,153 @@
 {% extends "base.njk" %}
 
 {% block content %}
-    <h1>
-        <a href="https://the-federation.info" style="color:inherit">
-            <img src="/images/the-federation.png">
-            <span style="padding-left: 30px;">
-                the federation - a statistics hub
-            </span>
-        </a>
-    </h1>
+<main>
+    <header class="main-header">
+        <div class="main-title">
+            <h1>The Federation</h1>
```

Comment:
The `h1` element could be an unstyled link to the site - this is nice for reloading the page while on it ;)